python/polycap.pyx

```diff
@@ -362,16 +451,79 @@ cdef class TransmissionEfficiencies:
     cdef polycap_transmission_efficiencies *trans_eff
```

Comment:
gezien het groot aantal member variables, en de soms zeer gelijkaardig property methods, raad ik u sterk aan de member variable names te prefixen met een underscore. Dat bevordert de leesbaarheid serieus. Sorry had dit in de andere classes ook al moeten doen!

Reply:
Zal integraal aanpassen :) Had me al afgevraagd wat een goede manier zou zijn om die properties en variable names uit elkaar te halen :)